data/strings/strings.txt

```diff
@@ -29439,6 +29623,9 @@ fa = دوباره نپرس
     sw = Alamisho %d
     zh-Hans = %d 個書籤
     zh-Hant = %d個書籤
+    fa:one = %d نشانه
+    af:other = %d نشانه‌ها
+
```

Comment:
One empty line is enough. Remove this one, please.

Reply:
fixed